extension/awsproxy/extension.go

```diff
@@ -16,17 +16,18 @@ import (
 )
 
 type xrayProxy struct {
-	logger *zap.Logger
-	config *Config
-	server proxy.Server
+	logger   *zap.Logger
```

Comment:
is this needed anymore with settings being part of the struct?

Reply:
not necessarily but I applied minimal changes, codeowners may choose to tidy afterwards or ask for more changes.